app/src/main/res/values/strings.xml

```diff
@@ -1,216 +1,304 @@
+    <string name="settings_hotwordPreference_summary">Enable \"Hotword Detection\" to interact with SUSI in hands-free mode</string>
     <string name="settings_micPreference_summary">Enable mic to give voice input</string>
-    <string name="settings_speechPreference_summary">Enable speech output only for speech input</string>
     <string name="settings_speechAlwaysPreference_summary">Enable speech output regardless of input type</string>
-    <string name="settings_hotwordPreference_summary">Enable \"Hotword Detection\" to interact with SUSI in hands-free mode</string>
-    <string name="settings_speechPreference_label">Speech output</string>
+    <string name="settings_speechAlways_key" translatable="false">speechOutputAlways</string>
     <string name="settings_speechAlways_label">Speech output always on</string>
-    <string name="settings_enterPreference_label">↵ to send</string>
-    <string name="setting_mic_input">Mic input</string>
-    <string name="setting_hotword_detection">Hotword Detection (Beta)</string>
-    <string name="setting_mic_key" translatable="false">micInput</string>
-    <string name="setting_hotword_key" translatable="false">hotword_detection</string>
-    <string name="settings_enterPreference_key" translatable="false">enterAsSend</string>
     <string name="settings_speechPreference_key" translatable="false">speechOutput</string>
-    <string name="settings_speechAlways_key" translatable="false">speechOutputAlways</string>
-    <string name="hello_world">Hello world!</string>
-    <string name="speech_prompt">Say something&#8230;</string>
+    <string name="settings_speechPreference_label">Speech output</string>
+    <string name="settings_speechPreference_summary">Enable speech output only for speech input</string>
+
     <string name="speech_not_supported">Sorry, your device doesn\'t support speech input</string>
+    <string name="speech_prompt">Say something…</string>
+
     <string name="tap_on_mic">Tap the mic icon to speak</string>
-    <string name="know_more">Know more</string>
 
     <!--Error Messages-->
     <string name="error_crop_not_supported">Sorry - your device doesn\'t support the crop action.</string>
-    <string name="error_occurred_try_again">An error occurred. Please try again.</string>
-    <string name="error_invalid_token">Invalid Access Token.</string>
-    <string name="error_internet_connectivity">Internet Connectivity Problem.</string>
     <string name="error_hotword">Sorry, hotword detection is not supported on your device.</string>
+    <string name="error_internet_connectivity">Internet Connectivity Problem.</string>
+    <string name="error_invalid_token">Invalid Access Token.</string>
+    <string name="error_occurred_try_again">An error occurred. please try again.</string>
     <string name="error_rating">Unable to rate skill. An error occurred.</string>
-    <string name="no_skill_name">Name not available</string>
+
```

Comment:
@arundhati24 this is the reason for the crash of the app on scrolling down the settingsfragment.
@iamareebjamal please review

Reply:
#1262

Reply:
Please add a new PR fixing the issue